src/app/containers/ATIAnalytics/index.test.jsx

```diff
@@ -73,7 +76,7 @@ describe('ATI Analytics Container', () => {
       canonical.default = mockCanonical;
 
       render(
-        <ContextWrap platform="canonical" pageType="article">
+        <ContextWrap platform="canonical" pageType="article" service="news">
```

Comment:
Whats the reason for some services being hardcoded and some not?

Reply:
The previous `<ContextWrap>` implementation had [`news`](https://github.com/bbc/simorgh/blob/754214bcc1894501552b257162714535f58a8218/src/app/containers/ATIAnalytics/index.test.jsx#L23) hardcoded as the service. We just adjusted this so we could [pass](https://github.com/bbc/simorgh/blob/993a50452ac3bf6b925b9024a56c66b4843edf23/src/app/containers/ATIAnalytics/index.test.jsx#L24-L25) in a service for the mocked context for `ukrainian`.